alpine-rails-buildkite/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM dkubb/alpine-rails-nginx
+
+# Remove booting the rails app
+RUN rm -r /etc/sv/rails /etc/service/rails
+
+MAINTAINER Dan Kubb <dkubb@fastmail.com>
+MAINTAINER Markus Schirp <mbj@schirp-dso.com>
+
+ENV BUILDKITE_AGENT_VERSION=edge                                \
+    BUILDKITE_BUILD_PATH=/opt/buildkite/builds                  \
+    BUILDKITE_HOOKS_PATH=/opt/buildkite/hooks                   \
+    BUILDKITE_BOOTSTRAP_SCRIPT_PATH=/opt/buildkite/bootstrap.sh
```

Comment:
Oh neat, I didn't know you could specify multiple ENV options like this.

Is there any advantage to this vs specifying one ENV per line? Normally docker creates one image per line in the config, but I thought maybe it collapsed consecutive ENV options into a single image.